codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1751,20 +1757,6 @@ removeHashObjectsByHashingVersion hashVersion =
       WHERE hash_version = :hashVersion
     |]
 
--- | Not used in typical operations, but if we ever end up in a situation where a bug
--- has caused the name lookup index to go out of sync this can be used to get back to a clean
--- slate.
-dropNameLookupTables :: Transaction ()
```

Comment:
The `(term|type)_name_lookup` tables are very old, and may or may not exist in codebases on share at the moment. I added a line to drop this table if it exists as part of the 11 -> 12 migration so we can now forget about them entirely.